src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 1; i <= 3; i++) {
```

Comment:
Start your iterator from `0` to `< 3` instead. Remember about `0` index, it can create a lot of problems with read/write operations with arrays.